GOVERNANCE.md

```diff
@@ -0,0 +1,69 @@
+# Cloud Native Buildpacks Governance
+This document defines the governance structure for Cloud Native Buildpacks. This project is committed to building an open, inclusive, productive and self-governing open source community that maintains high-quality tools for translating source code into OCI images. The community is governed by this document with the goal of defining how said community should work together to achieve this goal.
+
+## Repositories
+The following code repositories are governed by the Cloud Native Buildpacks community and maintained under the [buildpacks](https://github.com/buildpacks) organization.
+
+* [spec](https://github.com/buildpacks/spec)
+* [lifecycle](https://github.com/buildpacks/lifecycle)
+* [pack](https://github.com/buildpacks/pack)
+* [docs](https://github.com/buildpacks/docs)
+* [rfcs](https://github.com/buildpacks/rfcs)
+* [resources](https://github.com/buildpacks/resources)
```

Comment:
Should include https://github.com/buildpacks/samples and https://github.com/buildpacks/imgutil